client/src/main/java/io/pravega/client/netty/impl/ClientConnectionImpl.java

```diff
@@ -103,13 +107,18 @@ private void write(WireCommand cmd) throws ConnectionFailedException {
             public void operationComplete(ChannelFuture future) {
                 if (!future.isSuccess()) {
```

Comment:
`nettyHandler.setRecentMessage();` should be set here.

Reply:
Done